src/overdrive.js

```diff
@@ -4,6 +4,19 @@ import ReactDOM  from 'react-dom'
 const renderSubtreeIntoContainer = ReactDOM.unstable_renderSubtreeIntoContainer;
 const components = {};
 
+const prefix = styles => {
+    const propertiesToPrefix = ['transform', 'transformOrigin', 'transition'];
+    const prefix = 'Webkit';
+    const prefixedStyles = {};
+    Object.keys(styles).forEach(property => {
+        if (propertiesToPrefix.indexOf(property) >= 0) {
```

Comment:
Will need to be a nested loop then since we are checking an array of properties. I can't say if that is faster than indexOf to be honest. Happy to loop though. I can also convert these to traditional `for` loops which are typically a bit faster than `forEach`.